### PR TITLE
Add tutorial and minor fixes for navbar and generic error page

### PR DESCRIPTION
**Pull request by itsthejoker**
Relevant issue: N/a

## Description:

Adds a complete tutorial w/ some additional small fixes.

## Checklist:

- [x] Code Quality
- [x] Pep-8
- [ ] Tests (if applicable)
- [x] Success Criteria Met
- [ ] Inline Documentation
- [ ] Wiki Documentation (if applicable)

### CODE REVIEW

File: blossom/templates/app/partials/tutorial.partial [hunk 1]
```diff
@@ -1,37 +1,356 @@
-<div class="modal fade" id="tutorialModal" aria-hidden="true" aria-labelledby="exampleModalToggleLabel"
+{% load static %}
+
+<div class="modal fade" id="tutorialModal" aria-hidden="true" aria-labelledby="tutModalToggleLabel"
```
Comment (TimJentzsch) [inline blossom/templates/app/partials/tutorial.partial:3]: This file is huge, can we put each modal in a separate file to make it easier to understand?

File: blossom/templates/app/partials/tutorial.partial [hunk 2]
```diff
@@ -1,37 +1,356 @@
...
+            <div class="modal-header">
+                <h5 class="modal-title" id="tutModalToggleLabel5">Tutorial: Transcribing</h5>
+                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+            </div>
+            <div class="modal-body">
+                <p>
+                    Here is where the magic happens. Just write down what you see in the
+                    image! Remember that the goal here is that someeone who cannot access
+                    the image should be able to understand what is going on just by reading
+                    what you've written.
```
Comment (TimJentzsch) [inline blossom/templates/app/partials/tutorial.partial:187]: Maybe we should mention in some way that we use Markdown formatting?

But I'm not sure how to best introduce it, a Markdown tutorial would definitely be too big for this

**Review by TimJentzsch — CHANGES_REQUESTED** [review]
I haven't tested this locally, but the text looks good from what I have seen!

I'd just like to split up the template file again to make it easier to understand what's going on.

File: blossom/templates/app/partials/tutorial.partial [hunk 3]
```diff
@@ -1,37 +1,356 @@
...
+                    the green <strong>Load Template</strong> button to automatically copy
+                    it to the transcription window!<br>(Psst -- it will overwrite anything
+                    you've already put there, so do watch out!)
+                </p>
+                <div class="row">
+                    <div class="col-0 col-lg-2"></div>
+                    <div class="col-12 col-lg-8">
+                        <div class="text-center">
+                            <img src="{% static 'images/tutorial_5.png' %}" style="width: 100%"
+                                 alt="A screenshot showing a blue button labelled 'Reroll!'">
```
Comment (itsthejoker) [inline blossom/templates/app/partials/tutorial.partial:240]: wrong alt tag

File: blossom/templates/app/partials/tutorial.partial [hunk 4]
```diff
@@ -1,37 +1,356 @@
...
+                    anymore, we need to release it back so that other folks can give it a
+                    try. Clicking the red <strong>Reroll</strong> button will bring up the
+                    below screen.
+                </p>
+                <div class="row">
+                    <div class="col-0 col-lg-2"></div>
+                    <div class="col-12 col-lg-8">
+                        <div class="text-center">
+                            <img src="{% static 'images/tutorial_6.png' %}" style="width: 100%"
+                                 alt="A screenshot showing a blue button labelled 'Reroll!'">
```
Comment (itsthejoker) [inline blossom/templates/app/partials/tutorial.partial:292]: wrong alt tag

File: blossom/templates/app/partials/tutorial.partial [hunk 5]
```diff
@@ -1,37 +1,356 @@
...
+                    image! Remember that the goal here is that someeone who cannot access
+                    the image should be able to understand what is going on just by reading
+                    what you've written.
+                </p>
+                <div class="row">
+                    <div class="col-0 col-lg-2"></div>
+                    <div class="col-12 col-lg-8">
+                        <div class="text-center">
+                            <img src="{% static 'images/tutorial_4.png' %}" style="width: 100%"
+                                 alt="A screenshot showing a blue button labelled 'Reroll!'">
```
Comment (itsthejoker) [inline blossom/templates/app/partials/tutorial.partial:194]: wrong alt tag

**Review by TimJentzsch — APPROVED** [review]

**Pull request merged**